### ISSUE TITLE
Error in minification.costantFolding since 5.5.0

### ISSUE THREAD

**Issue by julien-f** [babel/babel#1687]
Stack trace:

```
json-rpc-peer/node_modules/babel/node_modules/babel-core/lib/babel/transformation/file/index.js:597
      throw err;
            ^
TypeError: src/index.js: boolean is not a function
    at NodePath.AssignmentExpression (json-rpc-peer/node_modules/babel/node_modules/babel-core/lib/babel/transformation/transformers/minification/constant-folding.js:31:13)
    at NodePath.call (json-rpc-peer/node_modules/babel/node_modules/babel-core/lib/babel/traversal/path/context.js:130:26)
    at NodePath.visit (json-rpc-peer/node_modules/babel/node_modules/babel-core/lib/babel/traversal/path/context.js:154:8)
    at TraversalContext.visitSingle (json-rpc-peer/node_modules/babel/node_modules/babel-core/lib/babel/traversal/context.js:92:12)
    at TraversalContext.visit (json-rpc-peer/node_modules/babel/node_modules/babel-core/lib/babel/traversal/context.js:104:19)
    at Function.traverse.node (json-rpc-peer/node_modules/babel/node_modules/babel-core/lib/babel/traversal/index.js:67:17)
    at NodePath.visit (json-rpc-peer/node_modules/babel/node_modules/babel-core/lib/babel/traversal/path/context.js:171:26)
    at TraversalContext.visitMultiple (json-rpc-peer/node_modules/babel/node_modules/babel-core/lib/babel/traversal/context.js:72:16)
    at TraversalContext.visit (json-rpc-peer/node_modules/babel/node_modules/babel-core/lib/babel/traversal/context.js:102:19)
    at Function.traverse.node (json-rpc-peer/node_modules/babel/node_modules/babel-core/lib/babel/traversal/index.js:67:17)
```

**Comment by sebmck** [comment]
What's the source of `src/index.js`?

**Comment by julien-f** [comment]
https://github.com/julien-f/js-json-rpc-peer/blob/b15a664a28f930c0173758a82644b03037ce6d8a/src/index.js

**Comment by sebmck** [comment]
Fixed as of 5.5.1, thanks! FYI, the `minification.constantFolding` transformer is **extremely** experimental so use with caution and please report any anomalies :)

**Comment by julien-f** [comment]
Thanks you for your responsiveness :)
